pl_pl.lang

```diff
@@ -108,104 +109,362 @@ gui.xaero_quick_use_6=Szybkie użycie 6
+gui.xaero_held_centered_position=Przedm. po wyśrodkowaniu
+gui.xaero_held_centered_position_screen_edges=Krawędzie
+gui.xaero_held_centered_position_hotbar=Pasek
+gui.xaero_minimap_frame=Styl ramki
+gui.xaero_minimap_frame_color=Kolor ramki
+gui.xaero_box_minimap_frame_color=Kolor ramki minimapy dla kolorowych stylów.
+gui.xaero_minimap_frame_default=Domyślny
+gui.xaero_minimap_frame_colored_thick=Kolorowy
+gui.xaero_minimap_frame_colored_thin=Kolorowy cienki
+gui.xaero_compass_north=N
+gui.xaero_compass_east=E
+gui.xaero_compass_south=S
+gui.xaero_compass_west=W
+gui.xaero_compass_scale=Skala kompasu na mapie
+gui.xaero_box_compass_scale=Skala liter kompasu (N, E, S, W), gdy kompas znajduje się na mapie.
+gui.xaero_minimap_compass_inside_frame=Na mapie
+gui.xaero_minimap_compass_on_frame=Na ramce
+gui.xaero_compass_color=Kolor cienia kompasu
+gui.xaero_box_compass_over_wp2=Renderowanie liter kompasu (N, E, S, W) nad waypointami na mapie. Działa to tylko wtedy, gdy kompas znajduje się na ramce.
+effect.xaerominimap.no_minimap=Bez minimapy
+effect.xaerominimap.no_entity_radar=Bez radaru bytów
+effect.xaerominimap.no_waypoints=Bez waypointów
+effect.xaerominimap.no_cave_maps=Bez tryb. jaskiniow.
+effect.xaerominimap.no_minimap_harmful=Bez minimapy
+effect.xaerominimap.no_entity_radar_harmful=Bez radaru bytów
+effect.xaerominimap.no_waypoints_harmful=Bez waypointów
+effect.xaerominimap.no_cave_maps_harmful=Bez tryb. jaskiniow.
+effect.xaerominimap.no_minimap_beneficial=Bez minimapy
+effect.xaerominimap.no_entity_radar_beneficial=Bez radaru bytów
+effect.xaerominimap.no_waypoints_beneficial=Bez waypointów
+effect.xaerominimap.no_cave_maps_beneficial=Bez tryb. jaskiniow.
+gui.xaero_radar_item_frames=Wyświet. kropek ramek na przedm.
+gui.xaero_show_dimension_name=Wyświetlanie bieżącego wymiaru
+gui.xaero_unknown_biome=Unknown Biome
+gui.xaero_light_value=Światło: %d
+gui.xaero_teleport_y_unknown=Nie możesz się przeteleportować do waypointa bez podanej współrzędnej Y. Dokonaj edycji tego waypointa, żeby móc się do niego teleportować.
+gui.xaero_box_waypoints_distance_scale2=Skala tekstów wyświetlających odległości od waypointów w grze.
+gui.xaero_box_waypoints_name_scale2=Skala tekstów wyświetlających nazwy waypointów w grze.
+gui.xaero_box_ingame_waypoint_icon_scale=Skala ikon waypointów renderowanych w grze.
+gui.xaero_ingame_waypoint_icon_scale=Skala ikon WP w grze
+gui.xaero_ingame_waypoint_close_scale=Skala bliskich WP w grze
+gui.xaero_box_ingame_waypoint_close_scale=Skala waypointów w grze, gdy znajdujesz się blisko nich. Aby zmienić skalę waypointów, które są daleko, użyj opcji: „Skala ikon WP w grze”, „Skala odległości od WP” i „Skala nazw waypointów”.
+gui.xaero_entity_icon_name_fallback=Nazwa bytu, gdy ikona zawiedzie
+gui.xaero_settings_search=Settings Search
```

Comment:
Not translated!

Reply:
Name of the text field used in narrations.

Reply:
For me, the accessibility narrator doesn't want to read this text aloud for some reason (it just says "edit box" when the search box is focused). Does it work for you?

Reply:
Tested on 1.16.5 and it works for me. Idk if there's a bug in newer versions but it's still a used line nonetheless.